lib/command.rb

```diff
@@ -37,4 +37,8 @@ def fail_level
     level = config.fetch("rubocop_fail_level", "")
     return "--fail-level #{level}" unless level.empty?
   end
+
+  def force_exclusion
+    return "--force-exclusion" if config["force_exclusion"]
```

Comment:
```suggestion
    force_exclusion = config.fetch("rubocop_force_exclusion", "")
    return "--force-exclusion" unless force_exclusion.empty? || force_exclusion == "false"
```

Reply:
@andrewmcodes I guess what we want to do with `config.fetch("rubocop_force_exclusion", "")` is grab the value and if empty return an empty string. However, when it's not empty it returns a boolean value and then force_exclusion.empty? fails due to an error.

```  
1) Command when config file exists returns built command
     Failure/Error: return "--force-exclusion" unless force_exclusion.empty? || force_exclusion == "false"

     NoMethodError:
       undefined method `empty?' for false:FalseClass
     # ./lib/command.rb:44:in `force_exclusion'
     # ./lib/command.rb:13:in `build'
     # ./spec/command_spec.rb:10:in `block (3 levels) in <top (required)>'
```
I changed `force_exclusion = config.fetch("rubocop_force_exclusion", "")` into ``force_exclusion = config.fetch("rubocop_force_exclusion", "").to_s` to overcome this issue. I hope that's okay.